keela-widgets/cameramanager.cpp

```diff
@@ -15,43 +15,44 @@
 #include "keela-pipeline/recordbin.h"
 #include "keela-widgets/plugin_utils.h"
 
-Keela::CameraManager::CameraManager(guint id, std::string pix_fmt, bool split_streams) : Bin("camera_" + std::to_string(
-            id)),
-    camera(Keela::get_video_test_source_name()) {
+Keela::CameraManager::CameraManager(guint id, std::string pix_fmt, bool split_streams) : Bin("camera_" + std::to_string(id)), camera(Keela::get_video_test_source_name()) {
     try {
         spdlog::info("Creating camera manager {}", id);
         this->id = id;
 
-        // Add all elements to the bin
-        add_elements(camera, caps_filter, transform,
-                     tee_main, tee_even, tee_odd,
-                     *trace_even, *presentation_even, snapshot_even,
-                     *trace_odd, *presentation_odd, snapshot_odd);
+        // Add all elements to the bin (except camera_stream_even which has deleted copy constructor)
+        add_elements(camera, caps_filter, transform, tee_main);
+
+        // @TODO: figure out how to do this more cleanly
+        // Add camera_stream_even manually due to deleted copy constructor from multiple inheritance
```

Comment:
Consider looking at how we use `recordbin` in the project. `recordbin` also has multiple inheritance but does not need to resolve ambiguous conversion when casting to Element &